<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportF32SetupDialog</class>
 <widget class="QDialog" name="ReportF32SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>258</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Параметры отчета</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата начала периода</string>
   </property>
  </widget>
  <widget class="CDateEdit" name="edtEndDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="CDateEdit" name="edtBegDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата окончания периода</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Подразделение</string>
   </property>
  </widget>
  <widget class="COrgStructureComboBox" name="cmbOrgStructure">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="CRBComboBox" name="cmbEventType">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип обращения</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpeciality">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>140</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Специальность</string>
   </property>
   <property name="buddy">
    <cstring>cmbSpeciality</cstring>
   </property>
  </widget>
  <widget class="CPersonComboBoxEx" name="cmbPerson">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>169</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblPerson">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>169</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Врач</string>
   </property>
   <property name="buddy">
    <cstring>cmbPerson</cstring>
   </property>
  </widget>
  <widget class="CRBComboBox" name="cmbSpeciality">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblAgeYears">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>200</y>
     <width>23</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>лет</string>
   </property>
   <property name="buddy">
    <cstring>edtAgeTo</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="edtAgeTo">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>200</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="edtAgeFrom">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblAge">
   <property name="geometry">
    <rect>
     <x>384</x>
     <y>612</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Во&amp;зраст с</string>
   </property>
   <property name="buddy">
    <cstring>edtAgeFrom</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblAge_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>197</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Во&amp;зраст с</string>
   </property>
   <property name="buddy">
    <cstring>edtAgeFrom</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblAgeTo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>16</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>по</string>
   </property>
   <property name="buddy">
    <cstring>edtAgeTo</cstring>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportF32SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportF32SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
